File: jmix-templates/content/project/flowui-application/src/main/java/${project_rootPath}/view/main/MainView.java
package ${project_rootPackage}.view.main;

import com.vaadin.flow.router.Route;
import io.jmix.flowui.app.main.StandardMainView;
import io.jmix.flowui.view.ViewController;
import io.jmix.flowui.view.ViewDescriptor;

@Route("")
@ViewController("${normalizedPrefix_underscore}MainView")
@ViewDescriptor("main-view.xml")
public class MainView extends StandardMainView {
}

File: jmix-templates/content/project/application-kotlin/src/main/kotlin/${project_rootPath}/view/main/MainView.kt
package ${project_rootPackage}.view.main

import com.vaadin.flow.router.Route
import io.jmix.flowui.app.main.StandardMainView
import io.jmix.flowui.view.ViewController
import io.jmix.flowui.view.ViewDescriptor

@Route("")
@ViewController(id = "${normalizedPrefix_underscore}MainView")
@ViewDescriptor(path = "main-view.xml")
open class MainView : StandardMainView()
